arangod/Aql/ConstrainedSortExecutor.cpp

```diff
@@ -209,7 +209,9 @@ std::pair<ExecutionState, NoStats> ConstrainedSortExecutor::produceRows(OutputAq
       return {ExecutionState::DONE, NoStats{}};
     }
     // We should never get here, as the following LIMIT block should never fetch
-    // more than our limit. He may only skip after that. Thus:
+    // more than our limit. He may only skip after that.
```

Comment:
Nitpick: in english the block is neuter.
```suggestion
    // more than our limit. It may only skip after that.
```